indico/modules/events/registration/controllers/display.py

```diff
@@ -329,6 +330,21 @@ def _process(self):
                                                is_restricted_access=self.is_restricted_access)
 
 
+class RHUploadRegistrationFile(UploadFileMixin, RHRegistrationFormRegistrationBase):
```

Comment:
I think `RHRegistrationFormBase` (and ditching REGISTRATION_REQUIRED) would make more sense - this endpoint will never be called in the context of an existing registration